server/starters/web-starter/src/main/java/com/oceanbase/odc/config/SecurityBeanConfiguration.java

```diff
@@ -59,8 +59,8 @@ public class SecurityBeanConfiguration {
 
 
     @Bean
-    public DaoAuthenticationProvider authenticationProvider() {
```

Comment:
this configuration is a common configuration, why should we move it?

Reply:
In fact, it will only work at WebSecurityConfiguration. So I made a custom implementation and moved to web-starts.